micro-core/src/test/java/com/oath/micro/server/plugin/Bean1.java

```diff
@@ -0,0 +1,7 @@
+package com.oath.micro.server.plugin;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Bean1 {
```

Comment:
nitpick could we rename this from Bean1?

Reply:
It doesn't do anything, there is a check to see if it is included and that is it.